programs/survival/revive_coral

```diff
@@ -0,0 +1,16 @@
+//allows player to revive dead corals with water bottles
+__on_player_right_clicks_block (player, item_tuple,hand, block,face,hitvec) -> (
+	l(item, count, nbt) = item_tuple || l('None', 0, null);
+	dead=l('dead_brain_coral_block','dead_bubble_coral_block','dead_fire_coral_block','dead_horn_coral_block','dead_tube_coral_block');
```

Comment:
you could just detect if blocks matches 'dead_\w+_coral_block` and remove 'dead_' from the block name when setting. A suggestion

Reply:
I have no clue how to do, could you please explain further?

Reply:
never mind, it took 15 mins of testing to figure out